File: frontend/src/services/webrtcConfig.js
/** Fallback STUN-only (cross-network needs server TURN — see /api/v1/call/ice-config). */
export const ICE_SERVERS = {
  iceServers: [
    { urls: 'stun:stun.l.google.com:19302' },
    { urls: 'stun:stun1.l.google.com:19302' },
  ],
};

export const MEDIA_CONSTRAINTS = { audio: true, video: false };

File: mobile-app/lib/services/webrtc_config.dart
/// Fallback STUN-only; production uses IceConfigService from backend.
class WebRtcConfig {
  static Map<String, dynamic> get iceServers => {
        'iceServers': [
          {'urls': 'stun:stun.l.google.com:19302'},
          {'urls': 'stun:stun1.l.google.com:19302'},
        ],
      };

  static const Map<String, dynamic> mediaConstraints = {
    'audio': true,
    'video': false,
  };
}
